frontend/www/js/omegaup/api.js

```diff
@@ -618,6 +618,17 @@ export default {
     googleLogin: _call('/api/session/googlelogin/'),
   },
 
+  Submission: {
+    latestSubmissions: _call('/api/submission/latestsubmissions', function(
```

Comment:
nit:

```suggestion
    latestSubmissions: _call('/api/submission/latestsubmissions/', function(
```

hay que incluir el `/` final siempre que sea posible.